modules/microshift-automation-example-rpm-systems.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-automation-example-rpm-systems_{context}"]
+= Automation integration example with systemd for RPM systems
```

Comment:
There might need to be a warning message for the user that this will not work on OSTree deployments because the filesystem in /usr/lib is read-only.